### ISSUE TITLE
Spider-Man MM works!

### ISSUE THREAD

**Issue by kaishajacob** [Nukem9/dlssg-to-fsr3#90]
I was testing out Frame Generation DLSS Tweaks, as the regular Frame Generation didn't work with MM, and it worked! 3080 ti laptop, and it is running a smooth 120 FPS with RTX on at 1440p and DLSS balanced. If the regular frame generation mod isn't working, give the dlss tweaks version a shot. this made it work with Alan Wake 2 and Spider-Man. Planning on testing Portal RTX currently, and will report if any issues occurs! One of the best mods ever.

**Comment by cichy45** [comment]
Could you tell what game version, mod version, mod install (dlss ini or nvngx.dll method?). Does HDR work? Are there any issues with UI artifacts etc?

**Comment by kaishajacob** [comment]
my monitor isn't hdr compatible so I can't say, but Miles Morales is the latest game version, mod version is 0.50, and it is the dlss tweaks/ini method. UI looked fine when swinging very fast. you can check if hdr works, as the latest version made it seem compatible.

**Comment by Jackich2** [comment]
> Could you tell what game version, mod version, mod install (dlss ini or nvngx.dll method?). Does HDR work? Are there any issues with UI artifacts etc? I will add this info to game compatibility list, maybe the autor will decide to merge that.
> 
> If you could provide info like below.
> 
> Game	Version	Mod version	Mod install	Works	Ingame HDR	Other issues
> Cyberpunk 2077	2.1	0.5	DLSSTweaks	✅	✅	Ghosting/artifact line at the bottem of the screen when driving fast

you see Ghosting Because of DLSS not because of FG mod, FG mod Only strengthens them

**Comment by MJ12MJ** [comment]
> Could you tell what game version, mod version, mod install (dlss ini or nvngx.dll method?). Does HDR work? Are there any issues with UI artifacts etc? I will add this info to game compatibility list, maybe the autor will decide to merge that.
> 
> If you could provide info like below.
> 
> Game	Version	Mod version	Mod install	Works	Ingame HDR	Other issues
> Cyberpunk 2077	2.1	0.5	DLSSTweaks	✅	✅	Ghosting/artifact line at the bottem of the screen when driving fast

It works on Hogwarts Legacy too, but i don't know about HDR. Works with 0.41 and 0.5.
https://ibb.co/2Fr20dg
https://ibb.co/4PxjRVk

**Comment by Jackich2** [comment]
> > It works on Hogwarts Legacy too, but i don't know about HDR. Works with 0.41 and 0.5.
> 
> How about visual artifacts? Is there any excessive ghosting or UI artifacts?

Did you read the above message?  The mod is not to blame for the ghosting, it is the vaunted dlss that gives the ghosting..Only a new version of dlss or a mod with fsr3 instead of dlss will fix this

**Comment by MJ12MJ** [comment]
> > It works on Hogwarts Legacy too, but i don't know about HDR. Works with 0.41 and 0.5.
> 
> How about visual artifacts? Is there any excessive ghosting or UI artifacts?

Didn't notice at the moment, UI looks good too.
Just a little bit blurry because we can't use DLSS Sharpening with FG or the game crash.

**Comment by cichy45** [comment]
> Did you read the above message? The mod is not to blame for the ghosting, it is the vaunted dlss that gives the ghosting..Only a new version of dlss or a mod with fsr3 instead of dlss will fix this

Not entirely true. Ratchet & Clank UI is butchered even without any upscale method enabled. This issue is purely per-game based. Yes, in some cases like CP2077 ghosting might be caused by DLSS upscale, but UI is separate thing.

> Didn't notice at the moment, UI looks good too (Using DLSS 3.5.10)
> Just a little bit blurry because we can't use DLSS Sharpening with FG or the game crash.
> And latest game version of course, i don't know about the version number but its the latest one released.

Thank you.

**Comment by Jackich2** [comment]
> CP2077 ghosting might be caused by DLSS upscale

Yes, and AW 2 ghosting - DLSS.

**Comment by FlowersPowerz** [comment]
i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales

**Comment by cichy45** [comment]
> i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales. I have an rtx 3090ti

Did you enable hardware gpu scheduling in windows settings?

**Comment by kaishajacob** [comment]
> > i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales. I have an rtx 3090ti
> 
> Did you enable hardware gpu scheduling in windows settings?

yeah what he said. make sure Hardware GPU Scheduling is on, and make sure you actually have and own DLSSTweaks, which is found on nexusmods. It's required to make frame generation work for both spider-man games.

**Comment by FlowersPowerz** [comment]
> > > i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales. I have an rtx 3090ti
> > 
> > 
> > Did you enable hardware gpu scheduling in windows settings?
> 
> yeah what he said. make sure Hardware GPU Scheduling is on, and make sure you actually have and own DLSSTweaks, which is found on nexusmods. It's required to make frame generation work for both spider-man games.

i have both but it remains grayed out :c

**Comment by FlowersPowerz** [comment]
> > > i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales. I have an rtx 3090ti
> > 
> > 
> > Did you enable hardware gpu scheduling in windows settings?
> 
> yeah what he said. make sure Hardware GPU Scheduling is on, and make sure you actually have and own DLSSTweaks, which is found on nexusmods. It's required to make frame generation work for both spider-man games.

What version of the game do you have? I found that I don't have the latest patch, maybe that's the problem, I will try to install the latest patch from steam.

**Comment by kaishajacob** [comment]
> > > > i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales. I have an rtx 3090ti
> > > 
> > > 
> > > Did you enable hardware gpu scheduling in windows settings?
> > 
> > 
> > yeah what he said. make sure Hardware GPU Scheduling is on, and make sure you actually have and own DLSSTweaks, which is found on nexusmods. It's required to make frame generation work for both spider-man games.
> 
> What version of the game do you have? I found that I don't have the latest patch, maybe that's the problem, I will try to install the latest patch from steam.

i have the latest version of both games.

**Comment by FlowersPowerz** [comment]
> > > > > i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales. I have an rtx 3090ti
> > > > 
> > > > 
> > > > Did you enable hardware gpu scheduling in windows settings?
> > > 
> > > 
> > > yeah what he said. make sure Hardware GPU Scheduling is on, and make sure you actually have and own DLSSTweaks, which is found on nexusmods. It's required to make frame generation work for both spider-man games.
> > 
> > 
> > What version of the game do you have? I found that I don't have the latest patch, maybe that's the problem, I will try to install the latest patch from steam.
> 
> i have the latest version of both games.

i have updated the game but the option is still grayed out. The popup of the activation of the mod shows, i don't know what the problem might be

**Comment by cichy45** [comment]
I have the same issue in my case, no idea why. I get popup, but not able to switch frame gen inside game.

**Comment by Marjaperse** [comment]
> > Could you tell what game version, mod version, mod install (dlss ini or nvngx.dll method?). Does HDR work? Are there any issues with UI artifacts etc? I will add this info to game compatibility list, maybe the autor will decide to merge that.
> > If you could provide info like below.
> > Game	Version	Mod version	Mod install	Works	Ingame HDR	Other issues
> > Cyberpunk 2077	2.1	0.5	DLSSTweaks	✅	✅	Ghosting/artifact line at the bottem of the screen when driving fast
> 
> It works on Hogwarts Legacy too, but i don't know about HDR. Works with 0.41 and 0.5. https://ibb.co/2Fr20dg https://ibb.co/4PxjRVk

Mine crashes before shader compile

**Comment by iKillThis** [comment]
Same issue here.
Tested both verison of the mod and I get the Popup on both but the setting is greyed out

**Comment by johnparker619** [comment]
FG is greyed out for me as well in Miles Morales. Latest version, GPU scheduling is on, tried both (the normal and the dlsstweaks method), I get the pop up everytime as well.

**Comment by johnparker619** [comment]
> > > > > > i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales. I have an rtx 3090ti
> > > > > 
> > > > > 
> > > > > Did you enable hardware gpu scheduling in windows settings?
> > > > 
> > > > 
> > > > yeah what he said. make sure Hardware GPU Scheduling is on, and make sure you actually have and own DLSSTweaks, which is found on nexusmods. It's required to make frame generation work for both spider-man games.
> > > 
> > > 
> > > What version of the game do you have? I found that I don't have the latest patch, maybe that's the problem, I will try to install the latest patch from steam.
> > 
> > 
> > i have the latest version of both games.
> 
> i have updated the game but the option is still grayed out. The popup of the activation of the mod shows, i don't know what the problem might be

FG is greyed out for me as well in Miles Morales. Latest version, GPU scheduling is on, tried both (the normal and the dlsstweaks method), I get the pop up everytime as well.

**Comment by FlowersPowerz** [comment]
> > > > > > > i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales. I have an rtx 3090ti
> > > > > > 
> > > > > > 
> > > > > > Did you enable hardware gpu scheduling in windows settings?
> > > > > 
> > > > > 
> > > > > yeah what he said. make sure Hardware GPU Scheduling is on, and make sure you actually have and own DLSSTweaks, which is found on nexusmods. It's required to make frame generation work for both spider-man games.
> > > > 
> > > > 
> > > > What version of the game do you have? I found that I don't have the latest patch, maybe that's the problem, I will try to install the latest patch from steam.
> > > 
> > > 
> > > i have the latest version of both games.
> > 
> > 
> > i have updated the game but the option is still grayed out. The popup of the activation of the mod shows, i don't know what the problem might be
> 
> FG is greyed out for me as well in Miles Morales. Latest version, GPU scheduling is on, tried both (the normal and the dlsstweaks method), I get the pop up everytime as well.

Do you have cracked version? Like flt?

**Comment by FlowersPowerz** [comment]
> > > > > > > i have installed the tweaks version, it shows me the popup, but the option to activate the frame gen in the game is grayed out for me in Miles Morales. I have an rtx 3090ti
> > > > > > 
> > > > > > 
> > > > > > Did you enable hardware gpu scheduling in windows settings?
> > > > > 
> > > > > 
> > > > > yeah what he said. make sure Hardware GPU Scheduling is on, and make sure you actually have and own DLSSTweaks, which is found on nexusmods. It's required to make frame generation work for both spider-man games.
> > > > 
> > > > 
> > > > What version of the game do you have? I found that I don't have the latest patch, maybe that's the problem, I will try to install the latest patch from steam.
> > > 
> > > 
> > > i have the latest version of both games.
> > 
> > 
> > i have updated the game but the option is still grayed out. The popup of the activation of the mod shows, i don't know what the problem might be
> 
> FG is greyed out for me as well in Miles Morales. Latest version, GPU scheduling is on, tried both (the normal and the dlsstweaks method), I get the pop up everytime as well.

works with patch 2.1012.0.0, if you don't have this patch it won't work. You can see the version moving the mouse on the exe, it will show you the version of the game

**Comment by Olenjan** [comment]
Marvel's Spider-Man: Miles Morales v2.209.0.0 DLSS frame gen grayed out.
Seems to be the game version issue.

**Comment by FlowersPowerz** [comment]
> Marvel's Spider-Man: Miles Morales v2.209.0.0 DLSS frame gen grayed out with dlssg-to-fsr3 v0.70 Seems to be the game version issue. Every other game tested is working as intended.

You have to update to 2.1012.0.0

**Comment by Olenjan** [comment]
> > Marvel's Spider-Man: Miles Morales v2.209.0.0 DLSS frame gen grayed out with dlssg-to-fsr3 v0.70 Seems to be the game version issue. Every other game tested is working as intended.
> 
> You have to update to 2.1012.0.0

Can confirm that latest version is working as intended.

**Comment by Gl3banya** [comment]
In spider-man fsr frame gen looks choppier than playing without it for some reason for me. All the other games like Cyberpunk, Witcher 3, Plague tale didn't have that issue.

**Comment by Hot44431** [comment]
same in spiderman miles morales fell choppier with it on fells like the game is stuck at 50hrz 

Cyberpunk Hogwarts and Alan wake 2 works perfectly

**Comment by Hot44431** [comment]
> Miles and Remastered both works smoothly for me. Do you have vsync forced on through the nvidia panel?

yes vsync is on from the Nvidia panel in game from from 55fps i go to stable 100 but game fells like its still 50fps

found a small fix manually changing the aspect ratio to 21.9 in the game setting gives me the real fluid feel of 100 fps but not im getting ghosting / artefacts on the sides and on the mini map when i switch the ratio back to auto game fells like 50 fps again losses al the fluid motion

**Comment by jb695** [comment]
When some of you have mentioned that the DLSSTweaks version of the mod is necessary for it to work with MM, do you mean for it to actually take effect visually? Or to be able to simply toggle the FG option on/off in game.

I ask because I have simply applied the non DLSSTweaks version of the mod to MM, and the FG option is selectable in game for me and my FPS jumps 50+ while also visually looking smoother. 

So I'm just wondering why it was mentioned that the DLSSTweaks version is necessary for the mod to work.

**Comment by kaishajacob** [comment]
> When some of you have mentioned that the DLSSTweaks version of the mod is necessary for it to work with MM, do you mean for it to actually take effect visually? Or to be able to simply toggle the FG option on/off in game.
> 
> I ask because I have simply applied the non DLSSTweaks version of the mod to MM, and the FG option is selectable in game for me and my FPS jumps 50+ while also visually looking smoother.
> 
> So I'm just wondering why it was mentioned that the DLSSTweaks version is necessary for the mod to work.

It's to be able to toggle the FG option on the game. Tried using the non DLSSTweaks version during the time of last week and it never gave me the option to enable the FG option. DLSStweaks just had better compatibility for my system, and can be different for other people.

**Comment by rakaki** [comment]
> > Miles and Remastered both works smoothly for me. Do you have vsync forced on through the nvidia panel?
> 
> yes vsync is on from the Nvidia panel in game from from 55fps i go to stable 100 but game fells like its still 50fps
> 
> found a small fix manually changing the aspect ratio to 21.9 in the game setting gives me the real fluid feel of 100 fps but im getting ghosting / artefacts on the sides/UI flickering of game interface and on the mini map when i switch the ratio back to auto game goes back to felling like 50 fps but the Flickering stops

also to me.FSP is higher ,but I do not feel more smoother.

**Comment by Hot44431** [comment]
> FSP

manualy change the aspect ratio of the game in the setting to anything else then auto and check if the fps fells smother

**Comment by jb695** [comment]
> > When some of you have mentioned that the DLSSTweaks version of the mod is necessary for it to work with MM, do you mean for it to actually take effect visually? Or to be able to simply toggle the FG option on/off in game.
> > I ask because I have simply applied the non DLSSTweaks version of the mod to MM, and the FG option is selectable in game for me and my FPS jumps 50+ while also visually looking smoother.
> > So I'm just wondering why it was mentioned that the DLSSTweaks version is necessary for the mod to work.
> 
> It's to be able to toggle the FG option on the game. Tried using the non DLSSTweaks version during the time of last week and it never gave me the option to enable the FG option. DLSStweaks just had better compatibility for my system, and can be different for other people.

Thanks for clarifying. Guess I was one of the lucky ones able to enable FG with the non DLSS Tweaks method.

**Comment by jnvillar** [comment]
I have spiderman v.2.209.0.0. I have installed dlss-to-fs3 tweaked mod and edited the `.ini` file. When I launch the game the fsr3 pop up appears but the option to enable dlss is greyed out. 

Should I download another version of the game?

**Comment by FlowersPowerz** [comment]
> * I have spiderman v.2.209.0.0.
> * I have installed dlss-tweaks and dlss-to-fs3 tweaked mod into the game folder.
> * I have edited the `.ini` file and put the path to the dlss-to-fs3 tweaked mod file
> 
> When I launch the game the fsr3 pop up appears but the option to enable dlss is greyed out.
> 
> Should I download another version of the game?

But why don't you read the previous answers? It is written a thousand times, you need the latest version the 2.1012.0.0, otherwise it doesn't work.
